README.md

```diff
@@ -222,8 +222,52 @@ labels:
 Scheduling Queries in Airflow
 ---
 
-Instructions for scheduling queries in Airflow can be found in this
-[cookbook](https://docs.telemetry.mozilla.org/cookbooks/bigquery-airflow.html).
+- bigquery-etl has tooling to automatically generate Airflow DAGs for scheduling queries.
+- Every query must be scheduled as part of a DAG that is specified in `dags.yaml`.
+  - New DAGs can be configured in `dags.yaml`, e.g., by adding the following:
+  ```yaml
+  bqetl_ssl_ratios:   # name of the DAG; must start with bqetl_
+    schedule_interval: 0 2 * * *    # query schedule
+    default_args:
+      owner: example@mozilla.com
+      start_date: '2020-04-05'  # YYYY-MM-DD
+      email: ['example@mozilla.com']
+      retries: 2    # number of retries if the query execution fails
+      retry_delay: 30m  
+  ```
+  - All DAG names need to have `bqetl_` as prefix.
+  - `schedule_interval` is either defined as a [CRON expression](https://en.wikipedia.org/wiki/Cron) or alternatively as one of the following [CRON presets](https://airflow.readthedocs.io/en/latest/dag-run.html): `once`, `hourly`, `daily`, `weekly`, `monthly`
+  - `start_date` defines the first date for which the query should be executed
```

Comment:
It would be worth noting here that Airflow will not automatically backfill older dates if `start_date` is set in the past, as readers may assume this.